src/struct_ls/red_black_constantcoef_gs.c

```diff
@@ -190,17 +185,14 @@ hypre_RedBlackConstantCoefGS( void               *relax_vdata,
 
                if (constant_coeff == 1)
                {
-                  Ai = hypre_CCBoxIndexRank(A_dbox, start);
-                  AApd = 1.0 / Ap[Ai];
-
                   hypre_RedBlackLoopInit();
 
 #define DEVICE_VAR is_device_ptr(xp,bp)
                   hypre_RedBlackConstantcoefLoopBegin(ni, nj, nk, redblack,
                                                       bstart, bni, bnj, bi,
                                                       xstart, xni, xnj, xi);
                   {
-                     xp[xi] = bp[bi] * AApd;
```

Comment:
Here we used multiplication because it was much faster than division at the time. I'm not sure this is a thing anymore.

Reply:
I think multiplication should be fine here because we're limited by memory bandwidth anyway.

I double-checked and `Ap[0]` is correct because we are in the (all) constant coefficients case here.

Reply:
Okay, I see now.  hypre_CCBoxIndexRank is always 0.  Thanks!